<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>supplier_interface</class>
 <widget class="QWidget" name="supplier_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nam_label">
     <property name="text">
      <string>Supplier name here</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="department_comboBox">
     <item>
      <property name="text">
       <string>...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lateshechka Solntsevo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lateshechka Korolev</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lateshechka Pervomayskaya</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delivery_button">
         <property name="text">
          <string>Подтвердить Доставку</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWebEngineView" name="map_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
